### ISSUE TITLE
Bug - Logout button not visible on responsive view for tablets

### ISSUE THREAD

**Issue by AlBundy33** [znuny/Znuny#255]
### Environment

- OS:           Server Linux, Client Android
- Browser:      Chrome
- Znuny version: latest version from https://github.com/juanluisbaptiste/docker-otrs (docker-compose.yml)

### Expected behaviour

usable navigation links in landscape mode
option to logout in portrait mode

### Actual behaviour

"user-icon" with Option to logout is not visible in portrait mode
cannot open sub-menus on top of the page (e.g. to create a new ticket - if I select an entry it did not open but it seems that i can move the entry to rearange the menu.

### How to reproduce

Testd on 10" android tablet.

see above

**Comment by hanneshal** [comment]
Have you tried on tapping the header?

![screencast 2022-06-04 14-09-14](https://user-images.githubusercontent.com/804939/171998310-ece627bc-3357-4d89-97f0-f37ebc1f4f97.gif)

**Comment by AlBundy33** [comment]
there are no buttons in portrait view
![Screenshot_20220604-143839_Chrome](https://user-images.githubusercontent.com/6294586/171999345-c136dfae-4702-475f-a3ce-c617d1cda3ee.jpg)

and in landscape mode i can not open the top-menus (..., Tickets, Admin, ...)
![Screenshot_20220604-143802_Chrome](https://user-images.githubusercontent.com/6294586/171999348-ea3de6c4-83dd-4421-852b-a635e65cbb42.jpg)

but at least I have the user icon to logout :-)

**Comment by hanneshal** [comment]
> Have you tried on tapping the header?

The icons are hidden on the mobile view. Tap on the header please. As shown in the screen recording.

Just a note: 
Your device shows a interesting behaviour. It should use responsive view or not. The switch from mobile view to desktop view, when in landscape makes no sense.

Nevertheless I can hit the "Profile" icon fine on my test tablets. One of them is an android using android 10 and GChrome.

Edit: Two questions:
1) What is your native resolution / or the exact name and model of the device you are using. 
2) Is there a "switch to mobile" link on the bottom of the page?

**Comment by AlBundy33** [comment]
aahhh - click on the header means "click on the header" not on the "menu-header" 😄 
ok this shows the logout button.


to the other issue: i have a samsung galaxy tab s5e and maybe a higher font size than usual.

**Comment by hanneshal** [comment]
>Is there a "switch to mobile" link on the bottom of the page?

Can you answer this to please. If so, please switch to responsive view. 

Sometime devices act like they not a mobile devices but mess up the click events. This seems to be the case here. 
Often we can not really do something about this. One tap on ticket should bring you to the ticket overview or open the ticket menu.

**Comment by AlBundy33** [comment]
there is no link on the bottom of the page
![Screenshot_20220604-163501_Chrome](https://user-images.githubusercontent.com/6294586/172007118-810d128b-d62d-499d-b089-8eabfe8c7992.jpg)

it seems that if i hit the right pixel on the top menu the the submenus will open but then i can not open the entries (e.g. to create a new ticket).
-> may it's much harder to hit the "right" pixels.

but in most of the cases i can't open the menus - instead the switch to a kind of movement-mode where a hook is shown on the right side if i have moved an entry to another position.
![Screenshot_20220604-164038_Chrome](https://user-images.githubusercontent.com/6294586/172007845-3b3f4d44-efa3-45a5-adcf-d0a31685c245.jpg)

**Comment by AlBundy33** [comment]
btw opening the menu on the user icon works with no issuee
![Screenshot_20220604-164151_Chrome](https://user-images.githubusercontent.com/6294586/172007903-3649106b-ec5d-4ee3-aed2-fe448f85974f.jpg)

**Comment by AlBundy33** [comment]
> Have you tried on tapping the header?
> 
> ![screencast 2022-06-04 14-09-14](https://user-images.githubusercontent.com/804939/171998310-ece627bc-3357-4d89-97f0-f37ebc1f4f97.gif) [ ](https://user-images.githubusercontent.com/804939/171998310-ece627bc-3357-4d89-97f0-f37ebc1f4f97.gif) ![screencast 2022-06-04 14-09-14](https://user-images.githubusercontent.com/804939/171998310-ece627bc-3357-4d89-97f0-f37ebc1f4f97.gif) [ ](https://user-images.githubusercontent.com/804939/171998310-ece627bc-3357-4d89-97f0-f37ebc1f4f97.gif)

off-topic: what software did you use to create the recordimg?

**Comment by hanneshal** [comment]
If this is the case it does not register the touch as touch but as "permanent click". Thats something we can't change at the moment. Sorry. If your device|software thinks it can it handle it correct, it should. If you can, maybe changing the user agent to something that says "Hey I'm a mobile device" could help. But I'm not a Android pro. 


The software is Monosnap

Regards
Johannes

**Comment by AlBundy33** [comment]
user agent is:
"Mozilla/5.0 (Linux; Android 11; SM-T720) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"
and if i select show desktop website in chrome
"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36"

**Comment by hanneshal** [comment]
Thanks. But I as I wrote, we can't do much about this. A "click" is a "click" not a "long click". 
I don't have this problem on one of my devices, this makes it nearly impossible to debug for me. Even an iPad Pro 12" shows the mobile view on landscape. Your has half the resolution and renders the page without mobile mode. 
The "desktop" mode is not made for mobile, so I don't bother about this not being fully usable.
Your mobile device should get the mobile view. This is the main error. But without a similar device and software I'm not able to help at the moment. 
 
 you can visit this site and paste a screenshot from the result  or copy the link (works for the second site)
 http://howbigismybrowser.com
 and
 https://www.whatsmybrowser.org

**Comment by AlBundy33** [comment]
![Screenshot_20220604-180725_Chrome](https://user-images.githubusercontent.com/6294586/172014901-a9077441-fed8-4239-b209-b06e3b52b33a.jpg)
![Screenshot_20220604-180813_Chrome](https://user-images.githubusercontent.com/6294586/172014907-e9f2b059-12ea-41d7-ba61-414d4303899f.jpg)

**Comment by AlBundy33** [comment]
do you have any ideas or is there really nothing you can do?

**Comment by hanneshal** [comment]
Hi, 
no, not really. As I already wrote. I tested it in Chrome and FF with your settings (in the simulator) and the clicks work fine. I don't have your specific device so, but even then I don't think this problem is related to the application, then to the browser. 

There are multiple components that come in play here. 

1) Check for Touch
https://github.com/znuny/Znuny/blob/dev/var/httpd/htdocs/js/Core.App.Responsive.js#L159

2) The actual screen size. Yours is XL, gathered from the resolution information, so te footer is "regular" in landscape mode.
https://github.com/znuny/Znuny/blob/dev/var/httpd/htdocs/js/Core.App.Responsive.js#L136

3) If Size is that large, reapply all the "original" code.
https://github.com/znuny/Znuny/blob/dev/var/httpd/htdocs/js/Core.Agent.Responsive.js#L184

We would need to check all of those, with your physical device. And thats the point where it gets a bit too far. It works fairly well with the devices (we as the team) have access to. It is not perfect, could be improved anytime. But it works. 

My wild guess is that the detection for touch fails some how. The browser API there is standard widely used. Which brings me back to the browser. 
It works on your device to, but "just" in portrait mode. Which lowers the priority on our list by a lot. 

If you have a good suggestion what we should do, let me know. Right now I don't see a good solution to resolve this. 

Sorry to disappoint
Johannes
